### PR TITLE
committing databases/week3 homework

### PR DESCRIPTION
**Pull request by fzscodes**

### CODE REVIEW

File: databases/week3/Homework_week3.sql [hunk 1]
```diff
@@ -0,0 +1,249 @@
...
+FROM meal
+WHERE price < 90;
+
+-- 2)Get meals that still has available reservations
+SELECT* 
+FROM Meal m
+JOIN Reservation rs
+ON m.id = rs.meal_id
+WHERE max_reservations > number_of_guests;
+
```
Comment (babakbandpey) [inline databases/week3/Homework_week3.sql:210]: Please correct the above query:
```

SELECT*
FROM   meal m
       JOIN reservation rs
         ON m.id = rs.meal_id
WHERE  max_reservations > number_of_guests
GROUP  BY rs.meal_id
HAVING Sum(rs.number_of_guests) < m.max_reservations; 

```

File: databases/week3/Homework_week3.sql [hunk 2]
```diff
@@ -0,0 +1,249 @@
...
+FROM meal
+LIMIT 5;
+
+-- 6)Get the meals that have good reviews
+SELECT m.title,m.description,m.location,m.max_reservations,m.price,r.meal_id,r.stars
+FROM Meal m
+JOIN Review r
+ON m.id = r.meal_id
+WHERE r.stars >= 4;
+
```
Comment (babakbandpey) [inline databases/week3/Homework_week3.sql:232]: Please correct the above query:
```

SELECT m.title,
       m.description,
       m.location,
       m.max_reservations,
       m.price,
       r.meal_id,
       Avg(r.stars)
FROM   meal m
       JOIN review r
         ON m.id = r.meal_id
GROUP  BY r.meal_id
HAVING Avg(r.stars) > 3;  
```

File: databases/week3/Homework_week3.sql [hunk 3]
```diff
@@ -0,0 +1,249 @@
...
+-- 7)Get reservations for a specific meal sorted by created_date
+SELECT * 
+FROM Reservation rs
+JOIN Meal m
+ON m.id = rs.meal_id
+WHERE m.id = 12
+ORDER BY rs.created_date DESC;
+
```
Comment (babakbandpey) [inline databases/week3/Homework_week3.sql:240]: Please notice the query below and correct accordingly

```
SELECT m.title, sum(rs.number_of_guests), rs.created_date
FROM Reservation rs
JOIN Meal m
ON m.id = rs.meal_id
-- WHERE m.id = 12
group by rs.meal_id, rs.created_date
ORDER BY rs.created_date DESC;

```

File: databases/week3/Homework_week3.sql (hunk 1 above)
Comment (fzscodes) [reply databases/week3/Homework_week3.sql:214]: Hi Babak..
I have implemented the feedback but I am getting an error on this query:

    SELECT*
FROM   meal m
       JOIN reservation rs
         ON m.id = rs.meal_id
WHERE  max_reservations > number_of_guests
GROUP  BY rs.meal_id
HAVING Sum(rs.number_of_guests) < m.max_reservations;

Comment (babakbandpey) [reply databases/week3/Homework_week3.sql:214]: Hello, sorry for the late answer. The mails have strangely arrive pretty late.

Would you please tell me what error you receive. I have no errors.
I have ran your SQLs and then the correction I suggested and it shows no errors.
Thank you

**Comment by fzscodes** [comment]
Hi Babak..I get an error in this query:
SELECT m.title
FROM   Meal m
       JOIN Reservation rs
         ON m.id = rs.meal_id
WHERE  m.max_reservations > rs.number_of_guests
GROUP  BY rs.meal_id
HAVING Sum(rs.number_of_guests) < m.max_reservations;

The error I get is:
Error Code: 1054. Unknown column 'm.max_reservations' in 'having clause'	0.0099 sec.
Could it be because I have an older version of mysql?

**Pull request merged**